src/cmd/initContainer.go

```diff
@@ -252,6 +252,23 @@ func initContainer(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if utils.PathExists("/usr/lib/rpm") {
+		logrus.Debug("Configuring RPM to ignore bind mounts")
+
+		rpmConfigString := `# Written by Toolbox
+# https://github.com/containers/toolbox
+
+%_netsharedpath /dev:/media:/proc:/sys:/tmp:/var/lib/flatpak
+`
+
+		rpmConfigBytes := []byte(rpmConfigString)
+		if err := ioutil.WriteFile("/usr/lib/rpm/toolbox",
+			rpmConfigBytes,
+			0644); err != nil {
+			return errors.New("failed to configure RPM to ignore bind mounts")
```

Comment:
```suggestion
			return fmt.Errorf("failed to configure RPM to ignore bind mounts: %v", err)
```
I believe, it is important to include the causing error.

Reply:
Good point! It should be `%w`, though, shouldn't it? So that the error can be unwrapped by the caller.